Update_user.css

```diff
@@ -20,36 +27,68 @@ body{
 
 nav{
 	width:100%;
-	background-color:white;
+	background-color:#0C59CF;
 	overflow:auto;
 }
 
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
```

Comment:
I change the margin for the link in the nav bar so the position is right for my link because the old margin was wrong because when I added the drop menu function my drop down menu column was shift and doesn't match. So I need to reposition of those column, but it only work if the type of screen was in a specific size which the the screen size of my laptop since I make it to be in with my screen size. So I decided to set a difference margin so I don't need to to reposition of the drop down menu column and it will be consistent for all screen.